src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -86,7 +88,8 @@
     public ValidationStringency VALIDATION_STRINGENCY = ValidationStringency.DEFAULT_STRINGENCY;
 
     @Option(doc = "Compression level for all compressed files created (e.g. BAM and GELI).", common=true)
-    public int COMPRESSION_LEVEL = BlockCompressedStreamConstants.DEFAULT_COMPRESSION_LEVEL;
+    // The default IntelDeflater is slow at levels > 1
+    public int COMPRESSION_LEVEL = 1;
```

Comment:
This comment is not accurate -- the Intel Deflater is faster than the JDK deflater by 15-45% on all compression levels:

![image](https://user-images.githubusercontent.com/798637/27750423-b75c88e0-5da5-11e7-92cc-d6e7ea9affb4.png)

The reason why we want to compress at level 1 is because reading from a level 1 compressed file is ~3x faster than reading from a level 5 compressed file (the former default), and only slightly larger:

![image](https://user-images.githubusercontent.com/798637/27750497-0745e77a-5da6-11e7-93b5-723bb0897c0a.png)